core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -319,6 +323,18 @@ import scala.util.control.NonFatal
       sender() ! handleMetadataRequest(req)
   }
 
+  def expectSettings: Receive = LoggingReceive.withLabel("expectSettings") {
```

Comment:
deal with the Future failing in this state

Reply:
Added.